src/explore-education-statistics-admin/src/modules/find-statistics/components/ReleaseContentAccordionSection.tsx

```diff
@@ -68,64 +61,76 @@ const ReleaseContentAccordionSection = ({
         },
       });
     },
-    [release.id, sectionId, sectionContent.length, attachContentSectionBlock],
+    [actions, release.id, sectionId, sectionContent.length],
   );
 
-  const updateBlockInAccordionSection = useCallback(
+  const updateBlock = useCallback(
     async (blockId, bodyContent) => {
-      await updateContentSectionBlock({
+      await actions.updateContentSectionBlock({
         releaseId: release.id,
         sectionId,
         blockId,
         sectionKey: 'content',
         bodyContent,
       });
     },
-    [release.id, sectionId, updateContentSectionBlock],
+    [actions, release.id, sectionId],
   );
 
-  const removeBlockFromAccordionSection = useCallback(
+  const removeBlock = useCallback(
     (blockId: string) =>
-      deleteContentSectionBlock({
+      actions.deleteContentSectionBlock({
```

Comment:
I see that just above, we `await` the call to `actions.updateContentSectionBlock()`.  Do we not need to do the same here?

Reply:
Done